### ISSUE TITLE
Issue in Learn more section of the Device Installation

### ISSUE THREAD

**Issue by yogeshasalkar** [MicrosoftDocs/memdocs#435]
Hi,
I have encountered the following error in the documentation.

**Hardware device installation by device identifiers:
Hardware device installation by setup classes:**
Above 2 settings' **Learn more** section should redirect to following policy CSP pages respectively,

**DeviceInstallation/PreventInstallationOfMatchingDeviceIDs
DeviceInstallation/PreventInstallationOfMatchingDeviceSetupClasses**

But it is redirecting to incorrect policy CSP pages as mentioned below

**DeviceInstallation/AllowInstallationOfMatchingDeviceIDs
DeviceInstallation/AllowInstallationOfMatchingDeviceSetupClasses**

Justification: I have set these settings using the MDM baseline in Intune and following registry keys are getting created.
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\PolicyManager\current\device\DeviceInstallation\PreventInstallationOfMatchingDeviceIDs
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\PolicyManager\current\device\DeviceInstallation\PreventInstallationOfMatchingDeviceSetupClasses

Thanks and Regards,
Yogesh Asalkar
Compliance Research Analyst

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8f0f4d5d-fdd1-22e7-6372-9916b199209f
* Version Independent ID: caeb9f8b-30ad-7f02-4740-56522b2f9b1b
* Content: [Intune security baselines settings for Windows 10 MDM - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/protect/security-baseline-settings-mdm-all?pivots=mdm-may-2019#device-installation)
* Content Source: [memdocs/intune/protect/security-baseline-settings-mdm-all.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/protect/security-baseline-settings-mdm-all.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* Technology: ****
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by yogeshasalkar** [comment]
Any reason for closure?

**Comment by Brenduns** [comment]
@yogeshasalkar  Hello Yogeshasalkar, This appears to have been an automated closing tied to a related content update PR to fix the docs for this.  I've not seen that type of linkage of PR to GitHub issue before today, so unexpected. Glad you commented to ask. 
 
On 6/29, I passed this issue to our development team to check for the correct links in the UI (as the content was using the Information link from the UI). They began internal tracking to resolve that (though I don't have an update on that here).  This issue was held open until we had confirmation of the correct links and a content update.  Recently, a PR with the correct links was submitted internally, which I've merged to the content for publication later today - which triggered the close. This then is the official confirmation that we're closing this GitHub issue with the doc update pending publication in a few hours, and the engineering team aware of and tracking any UI update necessary to fix the links from the information text to the correct CSP's.
